.pre-commit-config.yaml

```diff
@@ -0,0 +1,41 @@
+minimum_pre_commit_version: "2.9.0"
+ci:
+  autofix_prs: false
+exclude: ^src/vendor/
+repos:
+  - repo: https://github.com/Zac-HD/shed
+    rev: 0.9.5
+    hooks:
+      - id: shed
+        # TODO: bump to --py310-plus when Talon moves to Python 3.10.
+        args: [--refactor, --py39-plus]
+        types_or: [python, markdown, rst]
+  - repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
Might be a good idea to move pre-commit-hooks before shed? They should be faster I think; failing fast would probably be ideal.

Reply:
Done.  Fwiw, though, pre-commit doesn't short-circuit, so I'm not sure there's too much benefit

Reply:
ah, I suppose you want to run black after flynt anyway

Reply:
Yeah exactly. And as a matter of fact I did find it useful to be able to hit Ctrl-c when running locally if I noticed failure, so there is some benefit.

Thanks for all the helpful comments! I'm happy with how it turned out